src/collection.js

```
@@ -104,8 +104,13 @@ export default class Collection {
     });
   }
 
-  use(transfomer) {
-    this._transformers[transfomer.type].push(transfomer);
+  configure(config) {
+    if (typeof config !== "object") {
+      throw new Error("First argument of configure should be an object.");
+    }
+    if (config.remoteTransformers instanceof Array) {
+      this._transformers.remote = config.remoteTransformers;
+    }
```

We should probably raise instead of silently ignoring the passed value here.